### PR TITLE
discussed tweaks

### PR DESCRIPTION
**Pull request by SeanTConrad**

### CODE REVIEW

File: verinfast/cloud/aws/instances.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 import boto3
 
-import regions as r
+import cloud.aws.regions as r
```
Comment (aylusltd) [inline verinfast/cloud/aws/instances.py:6]: does `import .regions as r` work for you? I'd rather keep it a relative import, so we can continue testing individual files.

File: verinfast/cloud/azure/instances.py [hunk 2]
```diff
@@ -35,12 +36,11 @@ def get_instances(sub_id:str, path_to_output:str="./"):
         architecture = "x86_64"
         if "D2p" in hw or "E2p" in hw:
                 architecture="Arm"
-        print(status.serialize())
         my_instance = {
```
Comment (aylusltd) [inline verinfast/cloud/azure/instances.py:39]: oh, well it's literally missing id... that's on me.

File: verinfast/cloud/gcp/instances.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 from google.cloud import compute_v1
 
-from zones import zones
+from cloud.gcp.zones import zones
```
Comment (aylusltd) [inline verinfast/cloud/gcp/instances.py:6]: Same comment here about relative import `from .zones import zones`

File: verinfast/cloud/aws/instances.py (hunk 1 above)
Comment (SeanTConrad) [reply verinfast/cloud/aws/instances.py:6]: Nope:
`Traceback (most recent call last):
  File "/home/stconrad/verinfast/verinfast/./agent.py", line 56, in <module>
    from cloud.aws.instances import get_instances as get_aws_instances
  File "/home/stconrad/verinfast/verinfast/cloud/aws/instances.py", line 7
    import .regions as r
           ^
SyntaxError: invalid syntax`

If I understand it correctly, for imports, Python doesn't look in directories, it looks in the sys.path. The main script that was run gets its path added, but the imports (instances.py) in this case, don't know where they are.

Comment (aylusltd) [reply verinfast/cloud/aws/instances.py:6]: yeah, that's the frustration I have with testing... I want to test small pieces not wait for the whole thing to run. Duly noted and merging.

**Review by aylusltd — APPROVED** [review]

**Pull request merged**